src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -86,7 +87,8 @@ export default class AircraftInstanceModel {
         this.airlineId      = '';         // Airline Identifier (eg. 'AAL')
         this.airlineCallsign = '';
         this.flightNumber = '';         // Flight Number ONLY (eg. '551')
-        this.transponderCode = '';
+        // FIXME: temporary inclusion, should be removed before merging to develop
+        this.transponderCode = _random(1201, 5999);
```

Comment:
uniqueness is not enforced.  this is simply so I can see that stuff is working.  Frankly, this should be removed before we merge.